docs/admin/templates/managing-templates/devcontainers/devcontainer-security-caching.md

```diff
@@ -0,0 +1,62 @@
+# Dev container security and caching
+
+Ensure Envbuilder can only pull pre-approved images and artifacts by configuring
+it with your existing HTTP proxies, firewalls, and artifact managers.
```

Comment:
We have docs for how to do it with many of these in the envbuilder repo. May be a good way to call them out.